ftw/news/profiles/default/cssregistry.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<object name="portal_css" meta_type="Stylesheets Registry" autogroup="False">
+
+    <stylesheet title=""
+                cacheable="True"
+                compression="safe"
+                cookable="True"
+                enabled="1"
+                expression=""
```

Comment:
If you add a ftw.theming-scss-file for the icons anyway, we could a sass-version of the styling there too and disable this resource with the condition ```expression="not:request/HTTP_X_FTW_THEMING|nothing"```.